services/galley/test/integration/API/Util.hs

```diff
@@ -176,19 +176,23 @@ createBindingTeamWithQualifiedMembers num = do
   (tid, owner, users) <- createBindingTeamWithMembers num
   pure (tid, Qualified owner localDomain, map (`Qualified` localDomain) users)
 
-getTeams :: UserId -> TestM TeamList
-getTeams u = do
+getTeams :: UserId -> [(String, String)] -> TestM TeamList
```

Comment:
If you change this to `[(ByteString, Maybe ByteString)]`, you can use [query](https://hoogle.zinfra.io/file/wire-server/.stack-work/install/x86_64-linux/ab0cbe4775ad309acf98798387493c21d0a173639613f15518bc025dacb362f7/8.8.4/doc/bilge-0.22.0/Bilge-Request.html#v:query) instead of `mkQueryItem`.  (If you do, use `toByteString'` on the calling side instead of `show`.)